src/micro_http/src/server.rs

```diff
@@ -530,22 +530,27 @@ impl HttpServer {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use std::fs;
     use std::io::{Read, Write};
     use std::os::unix::net::UnixStream;
 
     use common::Body;
+    use utils::tempfile::TempFile;
+
+    fn get_path_to_socket() -> TempFile {
```

Comment:
nit: I would name this function -> `get_tmp_socket_file`.

My reasoning is the following: `get_path_to_socket` implies returning a path (which might be a string, a rust std `Path` etc.), but not actually a "file", even a `TempFile`. Moreover, we can not infer any information about the fact that we get a temporary resource (meaning that it will be unlinked once it gets out of scope) by just looking at a function call alone. Please let me know if it makes sense.